locales-pending/premium.ftl

```diff
@@ -199,21 +206,21 @@ dashboard-exposures-filter-reset = Reset
 
 dashboard-top-banner-scan-in-progress-title = Your scan is still in progress
 # Variables:
-# $data_breach_total_num is the total number of data breaches the user has.
+# $unresolved_exposures is the total number of unresolved exposures the user has.
 dashboard-top-banner-scan-in-progress-description =
-  { $data_breach_total_num ->
-      [one] We found { $data_breach_total_num } exposure so far, but we’re still scanning sites that sell your personal info. This should be done within 3 minutes.
-      *[other] We found { $data_breach_total_num } exposures so far, but we’re still scanning sites that sell your personal info. This should be done within 3 minutes.
+  { $unresolved_exposures ->
+      [one] We found { $unresolved_exposures } exposure so far, but we’re still scanning sites that sell your personal info. This should be done within 3 minutes.
+      *[other] We found { $unresolved_exposures } exposures so far, but we’re still scanning sites that sell your personal info. This should be done within 3 minutes.
   }
 dashboard-top-banner-scan-in-progress-fix-now-hint = You can refresh this page then, or start fixing your data breaches now.
 dashboard-top-banner-scan-in-progress-fix-later-hint = You can refresh this page then, or come back later.
 dashboard-top-banner-scan-in-progress-cta = See what’s ready now
 
 dashboard-top-banner-protect-your-data-title = Let’s protect your data
 # Variables:
-# $data_breach_total_num is the total number of data breaches the user has.
-# $data_broker_total_num is the total number of data brokers selling the user’s data.
-dashboard-top-banner-protect-your-data-description = We found your data in { $data_breach_total_num } data breaches and { $data_broker_total_num } sites selling your personal info. We’ll guide you step-by-step on how to fix it.
+# $data_breach_unresolved_num is the unresolved number of data breaches the user has.
+# $data_broker_unresolved_num is the unresolved number of data brokers selling the user’s data.
+dashboard-top-banner-protect-your-data-description = We found your data in { $data_breach_unresolved_num } data breaches and { $data_broker_unresolved_num } sites selling your personal info. We’ll guide you step-by-step on how to fix it.
```

Comment:
Not a regression, but I did have a drive-by l10n question about plurals...
Is it possible that `$data_breach_unresolved_num` or `$data_broker_unresolved_num` would be singular?

For example:
> "We found your data in _**1 data breaches**_ and _**1 sites**_ selling your personal info. We'll guide you step-by-step on how to fix it."

Reply:
This is a good one, Peter. I think we will address this in another PR (see slack)